### PR TITLE
[cexio] Adding implementation for  request deposit address

### PR DESCRIPTION
**Pull request by bryantharris**
Creating new pull request per request.

### CODE REVIEW

File: xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOAccountService.java [hunk 1]
```diff
@@ -40,8 +41,11 @@ public AccountInfo getAccountInfo() throws IOException {
 
   @Override
   public String requestDepositAddress(Currency currency, String... arguments) throws IOException {
+    CexIOCryptoAddress cryptoAddress = getCexIOCryptoAddress(currency.getCurrencyCode());
+    if ( cryptoAddress.getOk().equals("ok"))
+      return cryptoAddress.getData();
 
-    throw new NotAvailableFromExchangeException();
+    throw new IOException(cryptoAddress.getE() + ": " + cryptoAddress.getError());
```
Comment (timmolter) [inline xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOAccountService.java]: This should be an `ExchangeException`

File: pom.xml [hunk 2]
```diff
@@ -421,4 +421,4 @@
         </plugins>
     </build>
 
-</project>
\ No newline at end of file
+</project>
```
Comment (timmolter) [inline pom.xml]: This seems fishy.

**Comment by timmolter** [comment]
Could you create a new PR after syncing up? There are too many commits in this PR not related to your code changes.

File: pom.xml (hunk 2 above)
Comment (bryantharris) [reply pom.xml]: On the updated branch I didn't include this as it seems unnecessary.

**Comment by bryantharris** [comment]
Okay, created a new branch and tried to bring just the files across.  I apologize, I'm obviously not super advanced with GitHub.

After making the Abucoins implementation I also moved the error check/exception throw into the 'Raw' file as that seems like a better place.  Otherwise the implementation is the same.

**Pull request closed without merging**